src/User/ToggleActivatorResponseHandler.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Clearbooks\Labs\User;
+
+use Clearbooks\Labs\User\UseCase\ToggleActivator\Response;
+
+class ToggleActivatorResponseHandler implements UseCase\ToggleActivatorResponseHandler
+{
+    public function handleResponse( Response $response )
+    {
+        // TODO: Implement handleResponse() method.
```

Comment:
I'm not sure you should be creating this concrete class... I mean this is detail that we haven't got to